<ui version="4.0" >
 <class>AddNew</class>
 <widget class="QDialog" name="AddNew" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add New Item</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >:/divers/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupFiles" >
       <property name="title" >
        <string>Files</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="4" column="0" >
         <widget class="QRadioButton" name="translation" >
          <property name="text" >
           <string>Translation</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/treeview/images/linguist.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QRadioButton" name="resource" >
          <property name="text" >
           <string>Resources</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/treeview/images/qrc.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QRadioButton" name="dialog" >
          <property name="text" >
           <string>Dialog</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/treeview/images/designer.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="header" >
          <property name="text" >
           <string>Header</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/treeview/images/h.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QRadioButton" name="source" >
          <property name="text" >
           <string>Sources</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources/resources.qrc" >:/treeview/images/cpp.png</iconset>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Project or Scope:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>File Name :</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Location :</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboProjects" />
           </item>
           <item>
            <widget class="QLineEdit" name="filename" >
             <property name="text" >
              <string>newfile.cpp</string>
             </property>
             <property name="cursorPosition" >
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="location" />
             </item>
             <item>
              <widget class="QToolButton" name="directoryButton" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okButton" >
           <property name="text" >
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton" >
           <property name="text" >
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>126</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filename</tabstop>
  <tabstop>location</tabstop>
  <tabstop>directoryButton</tabstop>
  <tabstop>comboProjects</tabstop>
  <tabstop>source</tabstop>
  <tabstop>header</tabstop>
  <tabstop>dialog</tabstop>
  <tabstop>resource</tabstop>
  <tabstop>translation</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddNew</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
